handlers/oam_handler.go

```diff
@@ -200,7 +201,8 @@ func GETOAMRegisterHandler(typ string, rw http.ResponseWriter) {
 		res := OAM.GetWorkloads()
 
 		if err := enc.Encode(res); err != nil {
-			logrus.Error("failed to encode workload definitions")
+			h.log.Error(ErrWorkloadDefinition(err))
```

Comment:
There is a backendbuild error `undefined: h` this is because the 'h' is not defined here so consider wrting the line 196 as `func (h *Handler) ( GETOAMRegisterHandler(typ string, rw http.ResponseWriter) {`